### ISSUE TITLE
Can't delete "Click to join" Jitsi button from web or desktop app, only possible from the mobile app

### ISSUE THREAD

**Issue by bseclier** [RocketChat/Rocket.Chat#19070]
### Description:

It is impossible to delete the Jitsi button "Click to join" like any other message via web or desktop client. It is only possible to do it from the mobile app.

### Steps to reproduce:

1. Create a jitsi meeting will show the "Click to join" button
2. After the meeting, it is impossible to delete it like any other message

### Expected behavior:

I'd like to be able to delete it from web and desktop client like I can do with the mobile app.

### Actual behavior:

I can"t :)

### Server Setup Information:

- Version of Rocket.Chat Server: 3.7.0
- Operating System: CentOS 7
- Deployment Method: Docker, image tag 3.7.0
- Number of Running Instances: Several
- DB Replicaset Oplog: Enabled
- NodeJS Version: 12.18.4
- MongoDB Version: 3.4.24

**Comment by ankar84** [comment]
Confirm that all the way!

**Comment by geekgonecrazy** [comment]
I believe this might be a bug on mobile.  I don't know that the message is intended to be deletable since other system messages are not

**Comment by geekgonecrazy** [comment]
cc @diegolmello what do you think?

**Comment by diegolmello** [comment]
@geekgonecrazy yep, it looks like a bug on mobile, but I don't know.
Why disable this behaviour?

**Comment by ankar84** [comment]
> Why disable this behaviour?

Please don't disable it.

**Comment by geekgonecrazy** [comment]
Yeah I guess that’s the real question.. do we want users to be able to delete? If so we need to fix web.  If web is working as intended then we need to fix mobile.  🙈

**Comment by cperves** [comment]
Why not let choosing administrators to enable or not this behaviour throught settings
Then only creators of this button or enabled role would be able to delete or disable it

**Comment by bseclier** [comment]
Please, do not remove this bug/feature from the mobile app :)
I'd like to be able to remove this button like any other message. We have a lot of this things in the #general and I'm happy to prune them manually.

**Comment by Sing-Li** [comment]
Actually, having a button labelled "Click to join...", and when clicked always says "Sorry the meeting was long over", is just plain bad Ux.

**Comment by chaperonRose** [comment]
> Actually, having a button labelled "Click to join...", and when clicked always says "Sorry the meeting was long over", is just plain bad Ux.

I agree with that.

> I believe this might be a bug on mobile. I don't know that the message is intended to be deletable since other system messages are not

Maybe this button should rather be a standard message, not a system message. This way, no contradiction if user is able to delete the message with the link. When a user share a file for example, it's not a system message.
